config/dqedetailtab.csv

```diff
@@ -0,0 +1,4 @@
+name,public,descrip,params,return
+.dqe.constructcheck,1,Used to check if a construct exists,[construct: symbol;chktype: `table`variable`view`function],Return list boolean and results
```

Comment:
Is the comment supposed to read like this?
> Return list of boolean and results

I think it could be made clearer.